app/src/org/commcare/activities/LoginActivity.java

```diff
@@ -245,7 +246,13 @@ protected void onResumeFragments() {
             return;
         }
 
-        tryAutoLogin();
+        if (CommCareApplication._().isConsumerApp()) {
+            uiController.setUsername("t1");
+            uiController.setPasswordOrPin("123");
+            localLoginOrPullAndLogin(false);

```

Comment:
I see, `t1` / `123` are referenced in the build scripts. Not sure if it is worth it, but you can always inject them here from the build configs via `buildConfigField`